collector/collector_register/register.go

```diff
@@ -0,0 +1,51 @@
+package collector_register
+
+import (
+	"github.com/SUSE/sap_host_exporter/collector/dispatcher"
+	"github.com/SUSE/sap_host_exporter/collector/enqueue_server"
+	"github.com/SUSE/sap_host_exporter/internal/sapcontrol"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/log"
+)
+
+// RegisterOptionalCollectors register depending on the system where the exporter run the additional collectors
+func RegisterOptionalCollectors(webService sapcontrol.WebService) error {
+	enqueu_found := false
+	dispatch_found := false
```

Comment:
small typo here, also we use camelCase throughout the code, snake_case is only used exceptionally in package names:
```suggestion
	enqueueFound := false
	dispatcherFound := false
```